corehq/apps/es/case_search.py

```diff
@@ -129,6 +129,18 @@ def sort_by_case_property(self, case_property_name, desc=False):
             reset_sort=False
         )
 
+    def case_property_has_value(self, case_property_name):
+        return self.add_query(
```

Comment:
I'd strongly prefer using filters over queries unless you have a good reason, following the pattern described here:
https://commcare-hq.readthedocs.io/es_query.html#filtering